client/components/admin_form/index.coffee

```diff
@@ -30,3 +30,7 @@ module.exports = class AdminEditView extends Backbone.View
 
   ignoreUnsaved: ->
     window.onbeforeunload = ->
+
+module.exports.init = ->
+  new AdminEditView
+    el: $('body')
```

Comment:
Just curious what the reasoning behind wrapping this in an init was? I would check to make sure other pages that use `AdminEditView` aren't affected! ie, articles/:id/edit.